src/site/assets/css/home.css

```
@@ -0,0 +1,427 @@
+    *, *::before, *::after {
+        animation: none !important;
+        transition: none !important;
+    }
+    .reveal { opacity: 1; transform: none; }
+}
+
+/* ---------- injected daily content support -------------------------- */
+
+/* Live dot on the news card (markup from tools/inject-daily.js) */
+.daily-live-dot { animation: dailyPulse 2.4s ease-in-out infinite; }
+@keyframes dailyPulse { 0%, 100% { opacity: 1; } 50% { opacity: 0.3; } }
```

Stylelint (17.14.0)

[error] Expected keyframe name "dailyPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)